### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by sayrer** [google/protobuf-gradle-plugin#97]
0.7.8 or 0.8.0 would be nice.

**Comment by zhangkun83** [comment]
What particular change are you expecting?

**Comment by sayrer** [comment]
I want to use javalite and grpc 1.0.0-pre. The only way I can get it to work right is with 

```
buildscript {
    dependencies {
        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.0-SNAPSHOT'
    }
    repositories {
        mavenLocal() // built from source
    }
}
```

**Comment by zhangkun83** [comment]
0.7.7 also works. Use the following snippet:
```gradle
protobuf {
    protoc {
        artifact = 'com.google.protobuf:protoc:3.0.0'
    }
    plugins {
        javalite {
            artifact = 'com.google.protobuf:protoc-gen-javalite:3.0.0'
        }
    }
    generateProtoTasks {
        all().each { task ->
            task.builtins {
                remove java
            }
            task.plugins {
                javalite { }
            }
        }
    }
}
```

That said, a 0.8.0 release will happen soon, though it probably won't bring any new features, rather than a behavior change (#91) to align with the recently released protobuf 3.0.0.

**Comment by zhangkun83** [comment]
It's now released.
